cincinnati/src/lib.rs

```diff
@@ -583,11 +608,22 @@ impl<'a> Deserialize<'a> for Graph {
                     .map_err(|_| {
                         de::Error::invalid_value(serde::de::Unexpected::StructVariant, &self)
                     })?;
+
+                graph.conditional_edges.append(
+                    &mut conditional_edges
+                        .iter()
+                        .map(|ce| ce.clone())
+                        .collect::<Vec<ConditionalEdge>>(),
+                );
```

Comment:
```suggestion
                graph.conditional_edges.extend(conditional_edges);
```
to copy all elements from `conditional_edges` to `graph.conditional_edges`